magento/adobe-stock-integration
# Fix the Adobe Stock asset source icon url set by indexer

### Description (*)
This PR provides code and some possible business logic improvement for the enhancement media gallery image index.

**Issue related changes:**
Source image URL generates on the fly when the media gallery grid renders.

**Refactoring**
Remove source_icon_ur URL from the media grid. Remove the code which is not used for the new way of source image URL generation.

### Fixed Issues (if relevant)
#933 

### Manual testing scenarios (*)
Check the information at the #933 + test save the image and media grid reindex with the enhanced media gallery being turned off.
